velox/docs/functions/presto/url.rst

```diff
@@ -14,6 +14,36 @@ The URL extraction functions extract components from HTTP URLs (or any valid URI
 
 The extracted components do not contain URI syntax separators such as ``:`` , ``?`` and ``#``.
 
+Consider for example the below URI:
+
+.. code-block::
+
+    http://www.ics.uci.edu/pub/ietf/uri/?k1=v1#Related
+
+    scheme    = http
+    authority = www.ics.uci.edu
+    path      = /pub/ietf/uri/
+    query     = k1=v1
+    fragment  = Related
+
+
+Invalid URI's
+-------------
+Well formed URI's should not contain ascii whitespace and percent-encoded URI's should be followed by two hexadecimal
+digits following the percent character "%".
+
+.. code-block::
+
+    # URI with whitespace
+    SELECT url_extract_path('foo '); -- NULL (1 row)
```

Comment:
What do you want to show with this example? Are you trying to say that all URL functions return NULL when input URL is invalid? If so, please, state this directly.